src/Client.php

```diff
@@ -525,11 +525,18 @@ public function deleteOrganizationMetadata(int $organizationId, int $metadataId)
         $this->apiDelete($this->encode('/manage/organizations/%s/metadata/%s', $organizationId, $metadataId));
     }
 
-    public function createFeature(string $name, string $type, string $description, ?bool $canBeManageByAdmin = false, ?bool $canBeManagedViaAPI = true)
-    {
+    public function createFeature(
+        string $name,
+        string $type,
+        string $title,
```

Comment:
fakt chceme delat `major` jen kvuli jednomu (jeste optional) parametru? neslo by z toho vybrusli tak, ze to stricme na konec a dame default prazdny string?

Reply:
to jsem mel, ale nebylo to ani hezky ani prehledny - bylo to divny.
Idelani by byl Options object, ale ten by byl taky BC...

Reply:
BTW cemu to vadi?

Reply:
🤷 Jeste by slo pridat metodu `createFeatureWithTitle`, ale to mi prijde taky divny...